src/Authenticator/LoginHandler.php

```diff
@@ -512,7 +515,9 @@ public function getLogger(): ?LoggerInterface
     }
 
     /**
-     * Adds another option for the back URL to be returned from a current MFA session store
+     * Adds more options for the back URL - to be returned from a current MFA session store or from memory during this
+     * request. The latter is required as session store is cleared before the 'BackURL' key is utilised
```

Comment:
This docblock needs updating since you've removed the functionality it applies @ScopeyNZ :)